lib/src/charts/widgets/bar_chart.dart

```diff
@@ -27,6 +27,12 @@ class BarChart extends StatefulWidget {
   /// Defaults to true.
   final bool autoAnimate;
```

Comment:
Let's keep the naming convention consistent. 
If the other param is called `animateOnLoad` let's call this `animateOnUpdate`